packages/dara-components/package.json

```diff
@@ -72,17 +72,15 @@
     "prism-react-renderer": "^1.2.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-markdown": "^9.0.1",
     "react-plotly.js": "~2.6.0",
     "react-virtualized-auto-sizer": "1.0.6",
     "recoil": "^0.7.7",
-    "rehype-raw": "^6.1.1",
-    "remark-gfm": "^4.0.0"
+    "rehype-raw": "^6.1.1"
```

Comment:
I don't think we need the plugin as a direct dependency anymore?

Reply:
We still need rehypeRaw plugin as that is not added on dara-ui side, but the gfm plugin was removed

Reply:
Right, makes sense